doc/pipeable-operators.md

```diff
@@ -24,16 +24,6 @@ because an operator returns an `Observable`,
 not a `Promise`.
 There is now an `Observable.toPromise()`instance method.
 
-Because `throw` is a key word you could use `_throw` after `import { _throw } from 'rxjs/observable/throw'`.
-
-If the leading `_` bothers you (because a leading `_` typically means "_Internal - Do not use_"), you can do as follows:
-```
-import { ErrorObservable } from 'rxjs/observable/ErrorObservable';
-...
-const e = ErrorObservable.create(new Error('My bad'));
-const e2 = new ErrorObservable(new Error('My bad too'));
-```
```

Comment:
I think this whole part isn't necessary since in RxJS 6 you import one of these:
```
import { catchError } from 'rxjs/operators';
import { throwError } from 'rxjs';
```